### PR TITLE
Projects cards

### PR DESCRIPTION
**Pull request by nicocapalbo**

### CODE REVIEW

File: pfdp/css/variables.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
   --grey: #9b9b9b;
   --darkgrey: #606060;
   --darkergrey: #373632;
+  --pastel: #F2F0F0;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/variables.css]: **pastel** is not a good name for a variable - we don't understand that it's a very light grey
- maybe change by 'lightergrey' if it is lighter than the lightgrey color
- put it at the beginning of the grey list (as it's already by shade from light to dark)

File: pfdp/homepage.html [hunk 2]
```diff
@@ -92,7 +96,64 @@ <h1 class="white">Title here</h1>
     <!-- BLOCK TYPES OF FUNDRAISING EVENTS - ✅ GO - will just need to add some icons before the event name -->
 
     <div class="wrapper bg-white">
-      <div class="container-bm">fundraising events</div>
+      <div class="container-bm">
+        <div class="cards">
+          <div class="card">
+            <div class="card-top"></div>
+            <div class="pill">
+              <text class="pill-text">Anniversaire</text>
```
Comment (cecile-dzy-ncl) [inline pfdp/homepage.html]: <text> is not a valid HTML tag 😅

File: pfdp/homepage.html [hunk 3]
```diff
@@ -92,7 +96,64 @@ <h1 class="white">Title here</h1>
     <!-- BLOCK TYPES OF FUNDRAISING EVENTS - ✅ GO - will just need to add some icons before the event name -->
 
     <div class="wrapper bg-white">
-      <div class="container-bm">fundraising events</div>
+      <div class="container-bm">
+        <div class="cards">
+          <div class="card">
+            <div class="card-top"></div>
```
Comment (cecile-dzy-ncl) [inline pfdp/homepage.html]: here it will be a img tag more than a div with background image

File: pfdp/css/fundraisers.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
+.cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  /* flex-grow: 1; */
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: get rid of the comments

File: pfdp/css/fundraisers.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  /* grid-row-gap: 30px; */
+  /* grid-column-gap: 30px; */
+}
+
+.card {
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css:6]: - no margins on the cards
- no fixed width - they are going to take the full width of the grid / the number of fr in the grid
- better put an min-height here

File: pfdp/css/fundraisers.css [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+.card-top {
+  min-height: 50%;
+  width: 100%;
+}
+
+.pill {
+  position: absolute;
+  height: 75px;
+  width: 75%;
+  background-color: var(--red);
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: each pill should have a different bg-color (you indicate it by putting the css class on the pill html element)

File: pfdp/css/fundraisers.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  border-radius: 0px 300px 300px 0px;
+  top: 40%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: not sure here but this need a top and a translateY
i think it's :
- top: 50%
- transform: translateY(-50%)

File: pfdp/css/fundraisers.css [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+}
+.pill-text {
+  font-family: Montserrat;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: no need of font-family

File: pfdp/css/fundraisers.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+  font-style: normal;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: no need

File: pfdp/css/fundraisers.css [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  display: flex;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: pill-text doesn't have a child - why the display flex ?

File: pfdp/css/fundraisers.css (hunk 6 above)
Comment (cecile-dzy-ncl) [reply pfdp/css/fundraisers.css]: you can check the variable css file for all the bg-color class

File: pfdp/css/fundraisers.css [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+  align-items: center;
+  color: #FFFFFF;
+  padding-top: 20px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: why the `padding-top` ? if it is to center vertically the text in the pill, make the pill a flexbox + align-items to center

File: pfdp/css/fundraisers.css [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+  padding-left: 50px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css:40]: better put the padding on the parent than the text

File: pfdp/css/fundraisers.css [hunk 13]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.card-bottom {
+  height: 30%;
+  max-width: 100%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: no need for a width of max-width here - it's a div, so a display block and it's gonna take 100% of the width of the parent

File: pfdp/css/fundraisers.css [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+  padding: 15% 5% 10% 5%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: better in pixels here i think, not sure about responsiveness of this ?

File: pfdp/css/fundraisers.css [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+  background-color: var(--pastel);
+}
+
+.card-text {
+font-family: Montserrat;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: delete

File: pfdp/css/fundraisers.css [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+font-style: normal;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: delete

File: pfdp/css/fundraisers.css [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+font-weight: normal;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: delete

File: pfdp/css/fundraisers.css [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+font-size: 16px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: font-size of the body should already be at 16px (default font-size), so no need to put the font-size here

File: pfdp/css/fundraisers.css [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+line-height: 20px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: keep the line-height by default

File: pfdp/css/fundraisers.css [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+/* display: flex; */
+align-items: center;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: if the element is not a flexbox, the align-items will not work, no need of those two lines then

File: pfdp/css/fundraisers.css [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+@media screen and (max-width: 500px) {
+  .cards {
+    grid-template-columns: 1fr;
+  }
+  .card {
+    width: 273px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: no width here (see previous comment)

File: pfdp/css/fundraisers.css [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+    height: 338px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/fundraisers.css]: min-height here (same, see previous comment)

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

File: pfdp/css/variables.css (hunk 1 above)
Comment (nicocapalbo) [reply pfdp/css/variables.css]: changed to $lightergrey and moved to line 4

File: pfdp/homepage.html (hunk 2 above)
Comment (nicocapalbo) [reply pfdp/homepage.html]: changed **text** tag to **p** tag

File: pfdp/homepage.html (hunk 3 above)
Comment (nicocapalbo) [reply pfdp/homepage.html]: changed **div** tag for **img** tags

File: pfdp/css/fundraisers.css (hunk 4 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: comments deleted

File: pfdp/css/fundraisers.css (hunk 5 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css:6]: all done

File: pfdp/css/fundraisers.css (hunk 18 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed font size

File: pfdp/css/fundraisers.css (hunk 19 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed line-height

File: pfdp/css/fundraisers.css (hunk 20 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed align-items

File: pfdp/css/fundraisers.css (hunk 21 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed

File: pfdp/css/fundraisers.css (hunk 22 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: changed **height** to **min-height**

File: pfdp/css/fundraisers.css (hunk 7 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: done

File: pfdp/css/fundraisers.css (hunk 8 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed font family

File: pfdp/css/fundraisers.css (hunk 9 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed

File: pfdp/css/fundraisers.css (hunk 10 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: you are right, it should have been in the **pill** tag to be able to position the icon and pill text

File: pfdp/css/fundraisers.css (hunk 11 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: replaced for display:flex and align-items:center

File: pfdp/css/fundraisers.css (hunk 6 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: created classes for bg-colors in **variables.css**

File: pfdp/css/fundraisers.css (hunk 12 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css:40]: with the padding on the parent (the pill), it grows horizontally

File: pfdp/css/fundraisers.css (hunk 13 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: removed

File: pfdp/css/fundraisers.css (hunk 14 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: tweaked for responsiveness:
padding: 0 5% 0 5%;
@media --> padding: 20px 5% 0 5%;

File: pfdp/css/fundraisers.css (hunk 15 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: done

File: pfdp/css/fundraisers.css (hunk 16 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: done

File: pfdp/css/fundraisers.css (hunk 17 above)
Comment (nicocapalbo) [reply pfdp/css/fundraisers.css]: done

File: pfdp/css/fundraisers.css (hunk 12 above)
Comment (cecile-dzy-ncl) [reply pfdp/css/fundraisers.css:40]: try to add `box-sizing:border-box`

File: pfdp/css/variables.css [hunk 23]
```diff
@@ -16,3 +18,27 @@
   --shadow: 0px 10px 10px rgba(0, 0, 0, 0.1);
   --primary: var(--burgundy);
 }
+
+.burgundy {
```
Comment (cecile-dzy-ncl) [inline pfdp/css/variables.css]: `bg-burgundy`

File: pfdp/css/variables.css [hunk 24]
```diff
@@ -16,3 +18,27 @@
...
+  background-color: var(--burgundy);
+}
+
+.red {
```
Comment (cecile-dzy-ncl) [inline pfdp/css/variables.css]: `bg-red`

Comment (cecile-dzy-ncl) [reply pfdp/css/variables.css]: and so on for each color

background color class : `bg-nameofthecolor`
text color class : `nameofthecolor`

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

File: pfdp/css/variables.css (hunk 23 above)
Comment (nicocapalbo) [reply pfdp/css/variables.css]: change done

Comment (nicocapalbo) [reply pfdp/css/variables.css]: moved to utilities with the rest of bg color classes

File: pfdp/css/variables.css (hunk 24 above)
Comment (nicocapalbo) [reply pfdp/css/variables.css]: moved to utilities with the rest of bg color classes

**Pull request merged**